ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Translation.hs

```diff
@@ -17,7 +17,25 @@ import           Ouroboros.Consensus.TypeFamilyWrappers
 -------------------------------------------------------------------------------}
 
 data EraTranslation xs = EraTranslation {
+      -- | For each pair @(x, y)@ of subsequent eras, describe how to construct
+      -- the initial ledger state for @y@ from the last ledger state in @x@.
+      --
+      -- When ticking across an era boundary, the HFC will first invoke this and
+      -- then tick the resulting ledger state (in @y@) to the requested slot.
+      --
+      -- The resulting ledger state must summarize every relevant aspect of what
+      -- came before the new era. This is intentionally vague; for example,
+      -- ticking in @y@ might work rather differntly than in @x@, and so certain
+      -- aspects of the ticking logic of @x@ might need to happen as part of
+      -- 'translateLedgerState'. For a concrete example in Cardano, see
+      -- 'translateLedgerStateBabbageToConwayWrapper'.
       translateLedgerState   :: InPairs (RequiringBoth WrapLedgerConfig    (Translate LedgerState))       xs
+      -- | For each pair @(x, y)@ of subsequent eras, describe how to construct
+      -- the initial chain-dependent state for @y@ from the last chain-dep state
+      -- in @x@.
+      --
+      -- When ticking across an era boundary, the HFC will first invoke this and
+      -- then tick the resulting chain-dep state (in @y@) to the requested slot.
     , translateChainDepState :: InPairs (RequiringBoth WrapConsensusConfig (Translate WrapChainDepState)) xs
```

Comment:
Do we want to rename these? I am a bit unsure whether sth like `constructInitialLedgerState` are really more clear, but OTOH, `translateLedgerState` is maybe too generic?

Reply:
I would be in favor of renaming this to something more clear, that conveys not only in what context this will be used (as it is now with `translateX`) but also what it actually does (as it is in your proposed `constructInitialLedgerState`).

Reply:
Yes, I also think renaming to something much more concrete than "translate" is important.